docs/source/self_supervised_models.rst

```diff
@@ -0,0 +1,25 @@
+Self-supervised
+===============
+
+CPC (V2)
+--------
+PyTorch implementation of `Data-Efficient Image Recognition with Contrastive Predictive Coding <https://arxiv.org/abs/1905.09272>`_
+by (Olivier J. Hénaff, Aravind Srinivas, Jeffrey De Fauw, Ali Razavi, Carl Doersch, S. M. Ali Eslami, Aaron van den Oord).
+
+.. code-block:: python
+
+    from pl_bolts.models.self_supervised import CPCV2
```

Comment:
as it is v for the version I would rather use lower so it clear that V is not part of the shortcut.. `CPCv2`